lib/inventory_refresh/inventory_collection/helpers/questions_helper.rb

```diff
@@ -52,7 +52,7 @@ def saveable?
 
         # @return [Boolean] true if we are using a saver strategy that allows saving in parallel processes
         def parallel_safe?
-          @parallel_safe_cache ||= %i(concurrent_safe concurrent_safe_batch).include?(saver_strategy)
```

Comment:
Double checked that no one is referencing the ivar `@parallel_safe_cache` directly and no one is